src/pages/404.js

```diff
@@ -27,7 +28,9 @@ const NotFoundPage = () => {
 
             <p>Nothing to see here, go back to <a href={HOME.to}>POW!</a></p>
             <StaticImage
-              src="https://images.unsplash.com/photo-1554579653-17cdf3bfbca6?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MzkxfHxsZWdvJTIwYmF0bWFufGVufDB8fDB8fA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=400&q=60"
+              src="../images/Photo_by_Teo_Zac_unsplash_com_at_teo_2.jpeg"
```

Comment:
Is this correct @PaulieScanlon, I am not up to date with Gatsby image...

Reply:
Yes, this is correct according to: 
https://egghead.io/lessons/gatsby-process-images-and-render-them-using-gatsbyimage

by @laurieontech :

«We can pass it a source, in this case a URL, to an image. It can also take a relative path.»